.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          registry-url: https://npm.pkg.github.com/
+          scope: "@121cast"
+      - name: Build
+        run: |
+          npm run build
```

Comment:
Why do we need build when the publish builds?

Reply:
Just to validate commits/pull requests build without errors